### DISCUSSION TITLE
[A-L] (2023/24) Foglio 15 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#286] [Esercizi - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/c257c2b2-edcd-4e70-8831-eb37181a1ad6)

**Comment by Elia-Belli** [comment]
Per ottenere le basi di $U,V$ riduco a scala le matrici che hanno come colonne i vettori dei rispettivi Span, le riduco a scala e l'indice delle colonne dei pivot ottenute ci indicano i vettori linearmente indipendenti anche delle matrici di partenza:

$$\begin{pmatrix}1 & 0 & 2\\
1 & 1 & -1\\
1 & 0 & 2\\
1 & 1 & -1\end{pmatrix}\to\begin{pmatrix}1 & 0 & 2\\
0 & 1 & -3\\
0 & 0 & 0\\
0 & 0 & 0\end{pmatrix}\Rightarrow U=Span((1,1,1,1),(0,1,0,1))$$

$$\begin{pmatrix}2&2&2\\
1&3&-1\\
-2&-2&-2\\
-1&-3&1\end{pmatrix}\to\begin{pmatrix}2&2&2\\
0&2&-2\\
0&0&0\\
0 & 0 & 0\end{pmatrix}\Rightarrow V=Span((2,1,-2,-1),(2,3,-2,-3))$$

Facciamo la stessa cosa per i generatori di $U+V$ (ci conviene usare i vettori delle basi per averne 2 in meno):

$$\begin{pmatrix}1&0&2&2\\
1&1&1&3\\
1&0&-2&-2\\
1&1&-1&-3\end{pmatrix}\to\begin{pmatrix}1&0&2&2\\
0&1&-1&1\\
0&0&-4&-4\\
0&0&0&-4\end{pmatrix}\Rightarrow U+V=Span((1,1,1,1),(0,1,0,1),(2,1,-2,-1),(2,3,-2,-3))$$

Dalla cardinalità delle basi osserviamo la dimensione dei sottospazi e dal teorema della dimensione notiamo che $U\cap V=\underline 0$:

$$dim \ (U\cap V)=dim\ U+dim\ V-dim(U+W)=2+2-4=0$$

Quindi i sottospazi $U,V$ sono in somma diretta

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/141a97d4-3ce5-45a0-a6de-b6a2a59bee7d)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale

![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/3e0fdf9d-f5e2-45b3-a6ed-b44a585665aa)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8930ef39-5ca0-4a68-9425-65001a6bd200)
